osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/web/api/ThrottlingConfigController.java

```diff
@@ -143,7 +143,12 @@ public ResponseEntity<Short> createOrUpdateConfig(
   private boolean throttlingConfigNeedsUpdate(
       final org.opensmartgridplatform.throttling.entities.ThrottlingConfig throttlingConfigEntity,
       final ThrottlingConfig throttlingConfig) {
-    return throttlingConfigEntity.getMaxConcurrency() != throttlingConfig.getMaxConcurrency();
+    return throttlingConfigEntity.getMaxConcurrency() != throttlingConfig.getMaxConcurrency()
```

Comment:
Persoonlijk zou ik de vraag of er een update nodig is gaan stellen aan de entity zelf.

Reply:
Boyscouting done

Reply:
Ik zie nu pas dat het twee verschillende objecten zijn. Ben het eens met wat je zegt, maar je kan het ook omdraaien en de entiteit meegeven aan het config object. Verder geen showstopper voor mij, meer persoonlijke voorkeur. Hoewel ik wel denk dat bij uitbreiding van de condities dit fout gevoeliger is.

Reply:
Ik zie nu ook dat ze in verschillende modules zitten ... maakt de discussie ook weer anders. Lekker laten zo idd